From 6939f7af8a369dbd617c2551421588029215ce3a Mon Sep 17 00:00:00 2001
From: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>
Date: Fri, 19 Jul 2019 13:42:08 +0200
Subject: [PATCH 2/2] udev-extraconf: do not try to mount swap partitions

Swap is a special filesystem that cannot be mounted, so do not try to,
otherwise we will have service that tries and fails to mount it with
the following error:

systemd[1]: Mounting /run/media/nvme0n1p3...
mount[1229]: mount: /run/media/nvme0n1p3: unknown filesystem type 'swap'.

Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>
---
 meta/recipes-core/udev/udev-extraconf/mount.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/meta/recipes-core/udev/udev-extraconf/mount.sh b/meta/recipes-core/udev/udev-extraconf/mount.sh
index f655c7003d..3f3bd9ab1b 100644
--- a/meta/recipes-core/udev/udev-extraconf/mount.sh
+++ b/meta/recipes-core/udev/udev-extraconf/mount.sh
@@ -62,6 +62,8 @@ automount_systemd() {
     vfat|fat)
         MOUNT="$MOUNT -o umask=007,gid=`awk -F':' '/^disk/{print $3}' /etc/group`"
         ;;
+    swap)
+        ;;
     # TODO
     *)
         ;;
-- 
2.17.1

